raster/r.fill.stats/main.c

```diff
@@ -159,12 +159,12 @@ void print_weights_matrix(long int rows, long int cols)
             if (WEIGHTS[i][j] != -1.0) {
                 snprintf(weight_matrix_weight_buf, weight_matrix_line_length,
                          "%06.2f ", WEIGHTS[i][j]);
-                if (strlen(weight_matrix_weight_buf) >
+                /*if (strlen(weight_matrix_weight_buf) >
                     (weight_matrix_weight_length)) {
                     snprintf(weight_matrix_weight_buf,
                              weight_matrix_line_length, "[????] ",
                              WEIGHTS[i][j]);
-                }
+                }*/
```

Comment:
This appears to be a left-over development/debug code. I couldn't trigger it off and the extra snprinft() doesn't make much sense to me anyway (leave alone the format error leading me here).

Reply:
This has been around since [initial commit](https://trac.osgeo.org/grass/browser/grass-addons/grass6/raster/r.fill.gaps/main.c?rev=70778#L157) for G6.

Did anyone end up with `[????]` lines in printed matrix after `r.fill.stats -w ...` (or perhaps `r.fill.gaps`)??

Reply:
@benducke Maybe you can help out here?

Reply:
I prefer to keep this code active as long as there is a reason to test for `if (WEIGHTS[i][j] != -1.0)`

Reply:
Removed this particular fix from this PR.